x/project/internal/keeper/keeper.go

```diff
@@ -137,11 +139,11 @@ func (k Keeper) AddAccountToProjectAccounts(ctx sdk.Context, projectDid did.Did,
 }
 
 func (k Keeper) CreateNewAccount(ctx sdk.Context, projectDid did.Did,
-	accountId types.InternalAccountID) (auth.Account, sdk.Error) {
+	accountId types.InternalAccountID) (exported.Account, error) {
 	address := supply.NewModuleAddress(accountId.ToAddressKey(projectDid))
 
 	if k.AccountKeeper.GetAccount(ctx, address) != nil {
-		return nil, sdk.ErrInvalidAddress("Generate account already exists")
+		return nil, sdkerrors.Wrap(types.ErrInvalidAddress, "Invalid ProjectDid Address")
```

Comment:
Incorrect error message, please use the original one.

Reply:
corrected